terraform-dev/Makefile

```diff
@@ -1,6 +1,7 @@
 IMAGEURL := https://cloud-images.ubuntu.com/releases/bionic/release/ubuntu-18.04-server-cloudimg-amd64.img
 CPU_COUNT ?= 1
-DISK_SIZE ?= 15000000000
+ROOT_DISK_SIZE ?= 15000000000
+DATA_DISK_SIZE ?= 15000000000
```

Comment:
Do both disks need to be this size (doubling the amount of disk space used?)

Reply:
It won't use any additional disk space, unless needed. I actually set them both to be 50G just to be sure in the example below for 4 nodes:
```
193K /var/lib/libvirt/images/data-disk-0.qcow2
193K /var/lib/libvirt/images/data-disk-1.qcow2
193K /var/lib/libvirt/images/data-disk-2.qcow2
193K /var/lib/libvirt/images/data-disk-3.qcow2
899M /var/lib/libvirt/images/os-disk-0.qcow2
899M /var/lib/libvirt/images/os-disk-1.qcow2
899M /var/lib/libvirt/images/os-disk-2.qcow2
899M /var/lib/libvirt/images/os-disk-3.qcow2
12G /var/lib/libvirt/images/root-disk-0.qcow2
21G /var/lib/libvirt/images/root-disk-1.qcow2
20G /var/lib/libvirt/images/root-disk-2.qcow2
14G /var/lib/libvirt/images/root-disk-3.qcow2
```